src/testing/mocks.ts

```diff
@@ -230,12 +217,12 @@ export function mockCompilerSystem(): TestingSystem {
   return createTestingSystem();
 }
 
-export function mockDocument(html: string = null) {
+export function mockDocument(html: string) {
```

Comment:
Was the arg here intended not to be optional anymore? Looks like our internal implementations all provide an arg, but it this is also exported from our `testing` module. Also, `mockWindow` below had the same signature and that arg was kept optional.

Reply:
mmm tbh I don't remember why I did this! I'll change it so the type is `string | null`